### PR TITLE
modify docs with configure kerberos

### PR DESCRIPTION
**Pull request by liuxinzero07**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/docs/lakehouse/multi-catalog/hive.md [hunk 1]
```diff
@@ -177,6 +154,40 @@ CREATE CATALOG hive PROPERTIES (
...
+```sql
+(
+    'hadoop.security.authentication' = 'kerberos',
+    'hadoop.kerberos.keytab' = '/your-keytab-filepath/your.keytab',   
+    'hadoop.kerberos.principal' = 'your-principal@YOUR.COM',
+    'yarn.resourcemanager.principal' = 'your-rm-principal'
+);
+```
+
+请在所有的 BE、FE 节点下放置 krb5.conf 文件和 keytab 认证文件，keytab 认证文件路径和配置保持一致，krb5.conf 文件默认放置在 /etc/krb5.conf 路径。 hive.metastore.kerberos.principal 的值需要和所连接的 hive metastore 的同名属性保持一致，可从 hive-site.xml 中获取。 yarn.resourcemanager.principal 的值可从 yarn-site.xml 中获取。如若遇到查询 catalog 失败，可在 fe 启动脚本里增加 -Dsun.security.krb5.debug=true -Dsun.security.jgss.debug=true 进行 debug 。
```
Comment (wsjz) [inline docs/zh-CN/docs/lakehouse/multi-catalog/hive.md:170]: format configuration arguments

Comment (wsjz) [inline docs/zh-CN/docs/lakehouse/multi-catalog/hive.md:170]: format configuration arguments, just like the english docs

**Review by wsjz — APPROVED** [review]
LGTM

**Review by zy-kkk — APPROVED** [review]

**Comment by zy-kkk** [comment]
run buildall

**Pull request closed without merging**